src/parser/xasl_generation.c

```diff
@@ -3250,6 +3419,32 @@ pt_is_spec_node (PARSER_CONTEXT * parser, PT_NODE * tree, void *void_arg, int *c
   return tree;
 }
 
+/*
+ * pt_check_hashable () - check whether hashable or not
+ *   return:
+ *   parser(in):
+ *   tree(in):
+ *   void_arg(in):
+ *   continue_walk(in):
+ */
+static PT_NODE *
+pt_check_hashable (PARSER_CONTEXT * parser, PT_NODE * tree, void *void_arg, int *continue_walk)
+{
+  *continue_walk = PT_CONTINUE_WALK;
+
+  if (PT_IS_EXPR_NODE_WITH_OPERATOR(tree, PT_PRIOR))
+    {
+      ((HASHABLE *) void_arg)->is_PRIOR = true;
```

Comment:
consider readability for void_arg and check that it is guaranteed as "not null"

Reply:
I fixed void_arg for readability. and I didn't change about "not null'. because it is not check 'not null' in such functions related to walk tree.